examples/cropped_text.rs

```diff
@@ -0,0 +1,59 @@
+use embedded_canvas::Canvas;
+use embedded_graphics::{
+    pixelcolor::{Rgb555},
+    prelude::*,
+    primitives::{PrimitiveStyle, Rectangle},
+};
+use embedded_graphics_simulator::{
+    BinaryColorTheme, OutputSettingsBuilder, SimulatorDisplay, Window,
+};
+
+pub const DISPLAY_240P: Size = Size::new(320, 240);
+pub const DISPLAY_360P: Size = Size::new(480, 360);
+pub const DISPLAY_720P: Size = Size::new(1280, 720);
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    /* Algorithms:
+       1. Show a box with an outline
+       2. The box should contain 3 lines of text
+       3. Only the middle line will be fully shown while
+          the others will be partially visible
+    */
```

Comment:
There isn't any text written in this example, also I don't see the use of `canvas.crop()` method.
The example should show usage of `crop`

You can use `//` comments instead. `/* */` are really used